Makefile

```diff
@@ -1,5 +0,0 @@
-test-metrics-php-laravel:
```

Comment:
Removing this because I added it to help me debug the PHP Laravel app and we don't really need it around.